Visible/Non Visible Specifier Changes (#214)
BerkeleyLearnVerify/Scenic: docs/new.rst

	  Classes providing a default value for :prop:`heading` should instead provide a default value for :prop:`parentOrientation`.
	  Code like :specifier:`with heading 30 deg` should be replaced with the more idiomatic :specifier:`facing 30 deg`.

> 	* The :specifier:`visible` and :specifier:`not visible` now specify position uniformly randomly over all points that could result in a visible/not visible object (accounting for occlusion). This is different from previous versions, where the specifiers would simply place the center of an object either inside or outside the view region of the observing object.

Reviewer: ```suggestion
	* The :specifier:`visible` and :specifier:`not visible` specifiers now specify :prop:`position` uniformly randomly over all points that could result in the object being visible/not visible (accounting for occlusion). This is different from previous versions, where the specifiers would simply place the center of an object either inside or outside the view region of the observing object.
```
Actually I wonder if a higher-level description would be better, to make it clearer that the specifiers morally mean the same thing as before. Something like "The [specifiers] now take into account occlusion and the shapes of objects. In previous versions, they only checked whether the *center* of the object was visible/not visible, ignoring occlusion." People could still click through to the documentation of the specifier for the more precise definition. (We could even add "See :specifier:`visible` for details." if that would help.)

Also this should be under "backwards-incompatible semantics changes", not syntax changes.
